src/OpenGL.cpp

```diff
@@ -1327,6 +1327,10 @@ void OGLRender::_initExtensions()
 	m_bImageTexture = false;
 #endif
 	LOG(LOG_VERBOSE, "ImageTexture support: %s\n", m_bImageTexture ? "yes" : "no");
+	if (m_bImageTexture == false)
+	{
+		LOG(LOG_WARNING, "Image Textures not supported. Without OpenGL >= 4.3 or GLES >= 3.1, depth buffer emulation will not work");
```

Comment:
I think I've got my terminology wrong. I associate the fog effect in Beetle Adventure Racing with depth textures (from your blog post), and I forget what the name of the buffer used for that is (framebuffer texture?), but we definitely depend on image texture support for that type of fogging seeing as I get a black screen on those tracks that make use of it. Also there were a whole bunch of functions that simply return without doing any work if the image texture support wasn't available.

The N64 depth compare uses some other feature that my GL driver actually doesn't have, and thus I fail to run it with an error about something to do with memoryBarrier or something like that being called from a shader (I'm not at my computer right now otherwise I'd know the exact name).

I'll have a look later (maybe not the next couple of days. Flying on Friday) and clear up which features require which GL extensions and write down my evidence as to why.